### DISCUSSION TITLE
JetiLuaDFM with GPS-logger 3 by SM-Modellbau

### DISCUSSION THREAD

**Discussion by neffets88** [davidmcq137/JetiLuaDFM#16] [Q&A] [answered]
Hi David,
Thanks for sharing your work with us! Today I tried to use the GPS-Maps app on my ds24 in order to fly some triangles...
The App works great so far but does not show the right Speed values from my GPS-Logger (sm-Modellbau). It calls out the right "alt" value but the "speed" is always zero. It's not possible to start the Race. The position of the glider is also shown right.
Let me know if you need any logs or something for debugging. It's of course also possible that I did something wrong 😉
Greetings
Steffen

**Comment by davidmcq137** [comment, marked as answer]
thanks Steffen .. sorry I had not seen this when you first posted it. I hope we have this solved now.
